storage/src/backend/connection.rs

```diff
@@ -24,6 +24,11 @@ use nydus_api::http::{OssConfig, ProxyConfig, RegistryConfig};
 
 const HEADER_AUTHORIZATION: &str = "Authorization";
 
+thread_local! {
+    pub static LAST_LOGGING_TIME: RefCell<SystemTime> = RefCell::new(SystemTime::now());
```

Comment:
Is it necessary to use a global variable, how about naming `LAST_FALLBACK_AT`?

Reply:
Changed.